src/index.js

```diff
@@ -242,8 +242,11 @@ export default function buildPreset(context, opts = {}) {
     ]),
   );
 
-  useBuiltIns &&
-    plugins.push([transformPolyfillRequirePlugin, { polyfills, regenerator }]);
+  if (useBuiltIns === true) {
+    plugins.push([addUsedBuiltInsPlugin, { polyfills, regenerator, debug }]);
```

Comment:
The debug flag is used in the plugin => https://github.com/babel/babel-preset-env/blob/ebf77d82181ac0be44153cd2041a9cdd9a0c4e08/src/use-built-ins-plugin.js#L142